source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -13,13 +13,15 @@ Access Data From a Cursor
 Overview
 --------
 
-In this guide, you can learn how to access data using a cursor with the
+In this guide, you can learn how to access data using a **cursor** with the
 MongoDB Java driver. 
 
-Read operations that return multiple documents do not immediately return
-all values matching the query. Because a query can potentially match
-a large number of documents, we need to be able to access or store the
-matched documents. 
+A cursor is a tool for iterating over database results while only holding a
```

Comment:
```suggestion
A cursor is a tool that allows you to iterate over database results while only holding a
```
Suggestion: A cursor is used in other places outside of databases. I think it's more accurate to say it allows you to iterate.

Reply:
I think in this context this is a [database specific term](https://en.wikipedia.org/wiki/Cursor_(databases))

Reply:
In that case, I think you need to specify a database cursor since that's what your link says.